src/libs/brands.js

```diff
@@ -0,0 +1,146 @@
+    description: 'DataStage is a project powered by Terra for biomedical researchers to access data, run analysis tools, and collaborate.',
+    hostName: 'datastage.terra.bio',
+    docLinks: [
+      {
+        link: 'https://support.terra.bio/hc/en-us',
+        text: 'Find how-to\'s, documentation, video tutorials, and discussion forums'
+      }
+    ]
+  },
+  elwazi: {
+    name: 'eLwazi',
+    signInName: 'eLwazi',
+    welcomeHeader: 'Welcome to eLwazi',
+    description: 'The eLwazi Open Data Science Platform is a project powered by Terra for biomedical researchers to access data, run analysis tools, and collaborate.',
+    hostName: 'elwazi.terra.bio',
+    docLinks: [
+      {
+        link: 'https://support.terra.bio/hc/en-us',
+        text: 'Find how-to\'s, documentation, video tutorials, and discussion forums'
+      }
+    ]
+  },
+  firecloud: {
+    name: 'FireCloud',
+    signInName: 'FireCloud',
+    welcomeHeader: 'Welcome to FireCloud',
+    description: 'FireCloud is a NCI Cloud Resource project powered by Terra for biomedical researchers to access data, run analysis tools, and collaborate.',
+    hostName: 'firecloud.terra.bio',
+    docLinks: [
+      {
+        link: 'https://support.terra.bio/hc/en-us',
+        text: 'Find how-to\'s, documentation, video tutorials, and discussion forums'
+      },
+      {
+        link: 'https://support.terra.bio/hc/en-us/articles/360022694271',
+        text: 'Already a FireCloud user? Learn what\'s new.'
+      },
+      {
+        link: 'https://support.terra.bio/hc/en-us/articles/360033416912',
+        text: 'Learn more about the Cancer Research Data Commons and other NCI Cloud Resources'
+      }
+    ]
+  },
+  projectSingular: {
+    name: 'Project Singular',
+    signInName: 'Project Singular',
+    welcomeHeader: 'Welcome to Project Singular',
+    description: 'Project Singular is a project funded by Additional Ventures and powered by Terra for biomedical researchers to access data, run analysis tools, and collaborate.',
+    hostName: 'projectsingular.terra.bio',
+    docLinks: [
+      {
+        link: 'https://support.terra.bio/hc/en-us',
+        text: 'Find how-to\'s, documentation, video tutorials, and discussion forums'
+      }
+    ]
+  },
+  rareX: {
+    name: `The RARE${nonBreakingHyphen}X Data Analysis Platform`,
+    signInName: `the RARE${nonBreakingHyphen}X Data Analysis Platform`,
+    welcomeHeader: `Welcome to the RARE${nonBreakingHyphen}X Data Analysis Platform`,
+    description: `The RARE${nonBreakingHyphen}X Data Analysis Platform is a federated data repository of rare disease patient health data, including patient reported outcomes, clinical and molecular information. The platform is powered by Terra for biomedical researchers to access data, run analysis tools, and collaborate.`,
+    hostName: 'rare-x.terra.bio',
+    docLinks: [
+      {
+        link: 'https://rare-x.org/DataAnalysisPlatform-Documentation',
```

Comment:
This link returns a 404. Is that expected?

Reply:
Yes. I'll bring it to the PM's attention though, to make sure they're aware.